src/bsp/reboot.c

```diff
@@ -0,0 +1,46 @@
+#include <xss.h>
+#include <zephyr/drivers/pm_cpu_ops.h>
+#include <zephyr/kernel.h>
+
+#include "reboot.h"
+
+struct k_timer reboot_check_timer;
+struct k_work  perform_reboot_work;
+
+void perform_system_reboot(struct k_work* work)
+{
+    int                rc, value;
+    static const char* path = CONFIG_AOS_REBOOT_XEN_STORE_PATH;
+
+    rc = xss_read_integer(path, &value);
+    if (rc) {
+        return;
```

Comment:
Should we put some error here?

Reply:
in this case, we will receive error every time until someone write into this path